client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -207,6 +214,10 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				isRequestingSettings = isRequestingSettings || ( isRequestingJetpackSettings( state, siteId ) && ! jetpackSettings );
 			}
 
+			// Add custom `timezone_value` field
+			settings = { ...settings, timezone_value: getSiteTimezoneValue( state, siteId ) };
+			settingsFields.site.push( 'timezone_value' );
```

Comment:
It's a bit confusing to tie the two together, if we don't intend to save 'timezone_value'. Would it be possible to pass this through as another prop?